internal/kubernetes/operator/project/project.go

```diff
@@ -582,3 +601,83 @@ func buildAlertConfigurations(acProvider store.AlertConfigurationLister, project
 
 	return result, nil
 }
+
+func buildTeams(teamsProvider store.AtlasOperatorTeamsStore, orgID, projectID, projectName, targetNamespace string) ([]atlasV1.Team, []*atlasV1.AtlasTeam, error) {
```

Comment:
[q] Can you update the e2e test?

Reply:
Yes, a proper E2E test is required. I'll have to either significantly refactor current helpers or add a new one. I guess, I'll go the second way.